examples/defun.kl

```diff
@@ -19,12 +26,14 @@
 
 [example (const 'a 'b)]
 
+-- (last-stx foo bar baz) => baz
```

Comment:
This comment isn't really true. The issue is that `last-stx` is not a macro, it's a function. Its type would be `Syntax -> Syntax`, and its behavior is to return the last element in its argument. Here's a Haskell equivalent:

```
lastStx :: Syntax -> Syntax
lastStx (Syntax ctx (List xs)) = last xs
  where
    last (a:[]) = a
    last (a:d) = lastStx (Stx ctx d)
```

Reply:
removed; there is already an `[example (last-stx '(a b c d e f g))]` immediate afterwards anyway.